ygnmi/example_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ygnmi_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/openconfig/ygnmi/internal/exampleoc"
+	"github.com/openconfig/ygnmi/internal/exampleoc/root"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func ExampleLookup() {
+	c := initClient()
+	// Lookup a value at path.
+	path := root.New().Parent().Child()
+	val, err := ygnmi.Lookup(context.Background(), c, path.State())
+	if err != nil {
+		log.Fatalf("failed to lookup: %v", err)
```

Comment:
Go style says to generally capitalize everything other than fmt.Error messages. Can you capitalize all the log lines in this file?